todo_app/data/trello_items.py

```diff
@@ -35,34 +71,109 @@ def get_cards(id):
     return response
 
 
-def get_boardID(name):
-    searchUrl = baseUrl + "members/me/boards?fields=name"
+# get the lists on  board
+def get_boardLists(id):
+    searchUrl = f"{baseUrl}boards/{id}/lists"
 
     try:
-        response = requests.request("GET", searchUrl, params=query)
+        response = requests.request("GET", searchUrl, headers=headers, params=query)
         response.raise_for_status()
         response = response.json()
     except requests.exceptions.RequestException as exception:
         print(f"An error occured: {exception}")
         response = None
 
-    if response:
-        boardItem = next((obj for obj in response if obj["name"] == name), None)
-        if boardItem:
-            return boardItem["id"]
-        else:
-            return None
+    return response
+
+
+# get cards on a board
+def get_cards(id):
+    classItems = []
+    found = False
+
+    try:
+        # get all lists for the board
+        boardLists = get_boardLists(id)
+
+        # get the cards for each list
+        if len(boardLists) > 0:
+            for obj in boardLists:
+                cardsList = get_list_cards(obj["id"])
```

Comment:
I'm a big proponent of not over optimising, but currently your homepage loads fairly slowly for me and I suspect it's due to this line - we're making an extra call to the Trello API for every list (and my test board has about 10!)
One way we could cut that down quite substantially is to add the `cards=open` parameter to the `get_boardLists` function, so that we can just read those directly here:
```python
                cardsList = obj["cards"]
```